app/src/main/java/com/codesoom/assignment/MyHandler.java

```diff
@@ -0,0 +1,165 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+public class MyHandler implements HttpHandler {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final Map<Long, Task> taskMap = new ConcurrentHashMap<>();
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        final Method method = Method.valueOf(exchange.getRequestMethod());
+        final URI uri = exchange.getRequestURI();
+        final String path = uri.getPath();
+
+        final String[] pathArr = path.split("/");
+
+        String content = null;
+        final OutputStream outputStream = exchange.getResponseBody();
+
+        if (pathArr.length == 3) {
+            final Long id = Long.valueOf(pathArr[2]);
+
+            if (method.equals(Method.GET))
```

Comment:
이렇게 하면 `method`가 `null`인 경우에도 NPE가 던져지지 않을 것입니다.
```suggestion
            if (Method.GET.equals(method))
```

Reply:
전혀 생각하지 못한 부분이네요! 수정하겠습니다.

Reply:
Java 코딩시 흔하게 사용하는 기법이니 꼭 기억해 두세요. 앞으로도 자주 사용하게 되실 거에요.

Reply:
항상 유념하면서 코딩해야겠습니다